src/coreclr/pal/src/CMakeLists.txt

```diff
@@ -297,11 +299,21 @@ endif(CLR_CMAKE_TARGET_FREEBSD)
 if(CLR_CMAKE_TARGET_LINUX)
   # On Android, we don't need to link with gcc_s, pthread and rt
   if(NOT CLR_CMAKE_TARGET_ANDROID)
+    if(NOT CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
     target_link_libraries(coreclrpal
       gcc_s
       pthread
       rt
     )
+    else()
+    find_library(LZMA NAMES lzma)
```

Comment:
Same as above, this part I believe can be consolidated by putting this logic like that
```cmake
if(CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
  find_library(LZMA NAMES lzma)
  target_link_libraries(coreclrpal lzma)
endif()
```

Reply:
Thanks for your review!